modules/SpyUtils.js

```diff
@@ -4,6 +4,10 @@ import { isFunction } from './TestUtils'
 
 const noop = () => {}
 
+const engineSupportsConfigurableFunctionLength = (function () {
+  return Object.getOwnPropertyDescriptor(() => {}, 'length').configurable
```

Comment:
I've added that dependency as you suggested (and got rid of the IIFE as it wasn't really doing anything)